modules/dreamview/conf/vehicle_data.pb.txt

```diff
@@ -138,3 +138,15 @@ data_files {
   source_path: "perception_launch"
   dest_path: "/apollo/modules/perception/production/launch"
 }
+data_files {
```

Comment:
remove this, it's already added in https://github.com/ApolloAuto/apollo/blob/r5.5.0/modules/dreamview/conf/vehicle_data.pb.txt#L142